### PR TITLE
create user helper function that updates ideas

### PR DESCRIPTION
**Pull request by VladimirKarin**
https://vladimirkarin.atlassian.net/browse/FR-26

### CODE REVIEW

File: modals/ideas.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { getIdeas, setIdeas } = require('./ideas.js');
+
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+
+    const idea = ideas.find(idea => ideaId === idea.id);
+    const updateIdea = { ...idea };
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
```
Comment (fedosejev) [inline modals/ideas.js:9]: Typo

**Comment by fedosejev** [comment]
Typo in a branch name

File: modals/ideas.js (hunk 1 above)
Comment (VladimirKarin) [reply modals/ideas.js:7]: Updated the whole function

Comment (VladimirKarin) [reply modals/ideas.js:7]: Regarding branch name, I don't think that it's possible to edit the name? "creaste"...

Comment (fedosejev) [reply modals/ideas.js]: No, but you can create a new branch

**Pull request closed without merging**